docs/UserGuide.md

```diff
@@ -154,76 +281,92 @@ Format: `clear`
+* If multiple fields are specified, only jobs that match **all** the specified attributes will be returned.
 
-### Adding a comment on a candidate: `comment`
+Examples:
+* `search_job ti/Engineer d/C++` returns jobs whose titles contain `Engineer` and whose description contain `C++`.
+* `search_job v/10` returns jobs whose number of vacancies is 10.
 
-Leaves comments on important points to note down for individual candidates during the recruitment process. This overwrites existing comment (if any) and displays the resulting candidate.
+### Search for matching applications : `search_app`
 
-* `INDEX` must be within the range `1` to `n`, where `n` is the number of records in the database.
-* Any comment format is acceptable as long as comment is non-empty (i.e. user writes nothing in the comment field)
+Searches applications whose attributes match the specified attributes in the search criteria.
 
-Format: `comment INDEX m/COMMENT`
+Format: `search_app [e/EMAIL] [ti/TITLE] [s/STATUS]`
 
-Example:
-* `comment 3 m/Managed to solve every round 3 interview questions. He must be a strong candidate, potentially to be recruited as a quantitative research intern at Jane Street.`
+* At least one of the optional fields must be provided.
+* The search is case-sensitive, e.g. `engineer` will not match `Engineer`.
+* For title, partial words will be matched.
+* For email and status, only full words will be matched.
+* The search will fail if either email or status is in an invalid format.
+* If multiple fields are specified, only applications that match **all** the specified attributes will be returned.
 
+Examples:
+* `search_app e/alexyeoh@example.com s/PRESCREEN` returns applications from the candidate with email `alexyeoh@example.com` and with status `PRESCREEN`.
+* `search_app e/alexyeoh@example.com ti/Engineer` returns the application from candidate with email `alexyeoh@example.com` to the job with title containing `Engineer`.
+* `search_app ti/Software Engineer` returns applications to the job with title containing `Software Engineer`.
 
-### Tag a candidate: `tag`
+### Accessing by index: `get`
 
-Appends the tag or tags to a candidate's list of tags.
+Accesses candidates by **INDEX**. Here, **INDEX** refers to the index number of candidates shown in the displayed candidate list. The candidate index **must be** within the range from 1 to n, where n represents the **number of candidates** in the database.
 
-You can list any number of tags greater than 0, and all of them will be added to the specified **INDEX**. Here, **INDEX** refers to the index number of candidates shown in the displayed candidate list.
+Format: `get INDEX`
 
-Format: `tag INDEX [t/TAG]…​`
+* `INDEX` must be within the range `1` to `n`, where `n` is the number of records in the database.
 
-At least one tag must be provided.
+Example:
+* `get 24` returns the candidate with index 24.
 
-*Example 1* : `tag 24 t/smart`
+### Finding remaining vacancies: `slots left`
```

Comment:
should be slots_left with underscore in between

Reply:
Fixed